bpf/bpf_host.c

```diff
@@ -400,11 +400,6 @@ handle_ipv6_cont(struct __ctx_buff *ctx, __u32 secctx, const bool from_host,
 	}
 
 #if defined(ENABLE_IPSEC) && !defined(TUNNEL_MODE)
-	/* See IPv4 comment. */
-	if (from_proxy && info->tunnel_endpoint && encrypt_key)
-		return set_ipsec_encrypt(ctx, encrypt_key, info->tunnel_endpoint,
-					 info->sec_identity, true, false);
-
```

Comment:
This is in the packet path for `native-routing, connection was hijacked by the proxy`. Where would the encryption be applied now, in `to-netdev`?

Reply:
to-netdev will see:

Vxlan routing: 
- plain-text: host-to-host traffic
- plain-text-in-Vxlan: pod-to-pod traffic that requires further encryption
- ESP-in-Vxlan: old lingering encryption traffic

native routing:
- plain-text: pod-to-pod or host-to-any, that may require encryption
- ESP: old lingering traffic